ChatTTS/core.py

```
@@ -1,314 +1,756 @@
+                    False,
+                    self.device,
+                    use_decoder,
+                    params_infer_code,
+                )
+            )
+            wavs = self._decode_to_wavs(
+                result.hiddens if use_decoder else result.ids,
+                use_decoder,
+            )
+            result.destroy()
+            assert len(wavs), 1
```

Ruff (0.14.6)

Non-string literal used as assert message

(RUF040)